### PR TITLE
tests: add unit tests for run.go

### LINKED ISSUE #397: increase unit test coverage on run.go
Currently at 0%.

See #93.

### PR DESCRIPTION
**Pull request by devimc**
this patch increase the coverage of run.go

fixes #397

Signed-off-by: Julio Montes <julio.montes@intel.com>

### CODE REVIEW

File: run_test.go [hunk 1]
```diff
@@ -0,0 +1,531 @@
...
+)
+
+func TestRunCliAction(t *testing.T) {
+	assert := assert.New(t)
+
+	flagSet := flag.NewFlagSet("flag", flag.ContinueOnError)
+	flagSet.Parse([]string{"cc-runtime"})
+
+	// create a new fake context
+	ctx := cli.NewContext(&cli.App{Metadata: map[string]interface{}{}}, flagSet, nil)
```
Comment (jodh-intel) [inline run_test.go]: cute!

File: run_test.go [hunk 2]
```diff
@@ -0,0 +1,531 @@
...
+		{"", "", inexistentPath, inexistentPath, pidFilePath, false, runtimeConfig},
+		{"", "", inexistentPath, "", pidFilePath, false, runtimeConfig},
+		{"", "", consolePath, "", pidFilePath, false, runtimeConfig},
+		{"", bundlePath, consolePath, "", pidFilePath, false, runtimeConfig},
+		{testContainerID, inexistentPath, consolePath, "", pidFilePath, false, oci.RuntimeConfig{}},
+		{testContainerID, inexistentPath, consolePath, "", inexistentPath, false, oci.RuntimeConfig{}},
+		{testContainerID, bundlePath, consolePath, "", pidFilePath, false, oci.RuntimeConfig{}},
+		{testContainerID, inexistentPath, consolePath, "", pidFilePath, false, runtimeConfig},
+		{testContainerID, inexistentPath, consolePath, "", inexistentPath, false, runtimeConfig},
+		{testContainerID, bundlePath, consolePath, "", pidFilePath, false, runtimeConfig},
```
Comment (jodh-intel) [inline run_test.go]: wow ;)

**Comment by jodh-intel** [comment]
Just got that failing `TestRunContainerSuccessful` test under Travis to sort out.

**Comment by jodh-intel** [comment]
@devimc - argh! the dreaded Travis bug ;( You might need to close and re-open if a rebase doesn't cure it.

**Comment by devimc** [comment]
@jodh-intel yep, I rebased a lot of times, and seems like the bug still occurs

**Comment by devimc** [comment]
re-opened https://github.com/clearcontainers/runtime/pull/470

**Pull request closed without merging**